app/javascript/components/pf_aggregate_status_card.jsx

```diff
@@ -32,8 +32,8 @@ const PfAggregateStatusCard = ({
       </h2>
       <div className="card-pf-body">
         <p className="card-pf-aggregate-status-notifications">
-          { notifications.map(notification => (
-            <span key={notification} className="card-pf-aggregate-status-notification">
+          { notifications.map((notification,i)=> (
```

Comment:
Please move this into a separate PR as it is unrelated to it.

Reply:
I will move it to a new PR

Reply:
moved to a new PR https://github.com/ManageIQ/manageiq-ui-classic/pull/7295